mods/farming/api.lua

```diff
@@ -59,6 +59,11 @@ farming.hoe_on_use = function(itemstack, user, pointed_thing, uses)
 
 	if not minetest.setting_getbool("creative_mode") then
 		itemstack:add_wear(65535/(uses-1))
+		-- tool break sound
+		local wdef = itemstack:get_definition()
+		if itemstack:get_count() == 0 and wdef.sound and wdef.sound.breaks then
```

Comment:
After `itemstack:add_wear()` the itemstack is empty, and so `wdef` will get nothing. You need to put the line `local wdef = ....` above `itemstackadd_wear(...`

then it works.